src/Microsoft.AspNetCore.Razor.Language/BoundAttributeDescriptorExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public static class BoundAttributeDescriptorExtensions
+    {
+        public static string GetPropertyName(this BoundAttributeDescriptor descriptor)
+        {
+            descriptor.Metadata.TryGetValue(TagHelperBoundAttributeDescriptorBuilder.PropertyNameKey, out var propertyName);
+
+            return propertyName;
+        }
+
+        public static bool IsDefault(this BoundAttributeDescriptor descriptor)
```

Comment:
Ya I wasn't sold on `IsDefault` either but couldn't think of something better. Will rename.